next.config.mjs

```diff
@@ -1,5 +1,8 @@
 /** @type {import('next').NextConfig} */
 const nextConfig = {
+  env: {
+    STRIPE_SECRET_KEY: process.env.STRIPE_SECRET_KEY,
```

Comment:
@Bilalm354 Need to add this env var to vercel deployment settings

Reply:
donezo